<!DOCTYPE html>
<html lang="en">
  <head>
	<!--link href='http://fonts.googleapis.com/css?family=Roboto:400,300,100|Pontano+Sans|Open+Sans:300|Raleway' rel='stylesheet' type='text/css'-->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="description" content="">
    <meta name="author" content="">

    <link rel="shortcut icon" href="../../favicon.ico">

    <title>Linxi Liu</title>

    <!-- Bootstrap core CSS -->
    <link href="css/bootstrap.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/non-responsive.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

  <body>

    <!-- Fixed navbar -->
    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="#">Linxi Liu</a>
        </div>
        <div class="collapse navbar-collapse">
          <ul class="nav navbar-nav">
            <li><a href="index.html">Home</a></li>
			<li><a href="people.html">People</a></li>
            <li class="active"><a href="research.html">Research</a></li>
            <li><a href="teaching.html">Teaching</a></li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </div>

 <div class="container">

      <div class="page-header">

	<h3>Research Interests</h3>

	<ul style="line-height: 30px">
	  <li>Nonparametric Statistics</li>
	  <li>Multiple Hypothesis Testing</li>
	  <li>Bayesian Statistics</li>
	  <li>Statistical Machine Learning</li>  
	  <li>Statistical Genetics</li>
	</ul>

      </div>


	  <h3>Selected Preprints</h3>
		<p class="bibitem">He, Z., Chu, B., Yang, J., Gu, J., Chen, Z., <b>Liu, L.</b>, Morrison, T., Belloy, M. E., Qi, X., Hejazi, N., Mathur, M., Le Guen, Y., Tang, H., Hastie, T., Ionita-Laza, I., Cand&egrave;s, E. and Sabatti, C. <a href="https://www.biorxiv.org/content/10.1101/2024.02.28.582621v3"> Beyond guilty by association at scale: searching for causal variants on the basis of genome-wide summary statistics.</a> <i> Submitted.</i> </p>
			
		
	  	<h3>Publications</h3>
		<p class="bibitem">
		<b>Liu, L.</b> and Ma, L. <a href="https://proceedings.mlr.press/v247/liu24b.html">Spatial properties of Bayesian unsupervised trees.</a> 
		<i>Proceedings of Thirty Seventh Conference on Learning Theory (COLT), PMLR 247: 3556–3581, 2024.</i>
		</p>
		
		<p class="bibitem">
		Feng, H., Lu, X.J., Maji, S., <b>Liu, L.</b>, Ustianenko, D., Rudnick, N. D. and Zhang, C. 
		<a href="https://www.nature.com/articles/s41467-024-46429-y">Structure-based prediction and characterization of photo-crosslinking in native protein-RNA complexes.</a> 
		<i>Nature Communications, 15(1): 2279, 2024.</i>
		</p>
		
		<p class="bibitem">
		Yang, Z., Wang, C., <b>Liu, L.</b>, Khan, A., Lee, A., Vardarajan, B., Mayeux, R., Kiryluk, K. and Ionita-Laza, I. 
		<a href="https://doi.org/10.1038/s41588-023-01392-0">CARMA is a new Bayesian model for fine-mapping in genome-wide association meta-analyses.</a> 
		<i>Nature Genetics, 55(6): 1057–1065, 2023.</i>
		</p>
		
		<p class="bibitem">
		<b>Liu, L.</b>, Li, D. and Wong, W. H. 
		<a href="http://jmlr.org/papers/v24/20-060.html">Convergence rates of a class of multivariate density estimation methods based on adaptive partitioning.</a> 
		<i>Journal of Machine Learning Research, 24(50): 1–64, 2023.</i>
		</p>
		
		<p class="bibitem">
		Ma, S., Wang, C., Khan, A., <b>Liu, L.</b>, Dalgleish, J., Kiryluk, K., He, Z. and Ionita-Laza, I. 
		<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02864-6">BIGKnock: fine-mapping gene-based associations via knockoff analysis of biobank-scale data.</a> 
		<i>Genome Biology, 24(1): 24, 2023.</i>
		</p>
		
		<p class="bibitem">
		He, Z., <b>Liu, L.</b>, Belloy, M. E., Le Guen, Y., Sossin, A., Liu, X., Qi, X., Ma, S., Wyss-Coray, T., Tang, H., Sabatti, C., Cand&egrave;s, E., Greicius, M. and Ionita-Laza, I. 
		<a href="https://www.nature.com/articles/s41467-022-34932-z">GhostKnockoff inference empowers identification of putative causal variants in genome-wide association studies.</a> 
		<i>Nature Communications, 13(1): 7209, 2022.</i>
		</p>
		
		<p class="bibitem">
		<b>Liu, L.</b>, Meng, Y., Wu, X., Ying, Z. and Zheng, T. 
		<a href="https://doi.org/10.1080/10618600.2022.2051530">Log-rank-type tests for equality of distributions in high-dimensional spaces.</a> 
		<i>Journal of Computational and Graphical Statistics, 31(4): 1384–1396, 2022.</i>
		</p>
		
		<p class="bibitem">
		Yang, Y., Wang, C., <b>Liu, L.</b>, Buxbaum, J., He, Z. and Ionita-Laza, I. 
		<a href="https://doi.org/10.1016/j.ajhg.2022.08.013">KnockoffTrio: A knockoff framework for the identification of putative causal variants in genome-wide association studies with trio design.</a> 
		<i>The American Journal of Human Genetics, 109(10): 1761–1776, 2022.</i>
		</p>
		
		<p class="bibitem">
		He, Z., Le Guen, Y., <b>Liu, L.</b>, Lee, J., Ma, S., Yang, A. C., Liu, X., Rutledge, J., Losada, P. M., Song, B., Belloy, M. E., Butler, R. R., Longo, F. M., Tang, H., Mormino, E. C., Wyss-Coray, T., Greicius, M. and Ionita-Laza, I. 
		<a href="https://doi.org/10.1016/j.ajhg.2021.10.009">Genome-wide analysis of common and rare variants via multiple knockoffs at biobank scale, with an application to Alzheimer disease genetics.</a> 
		<i>The American Journal of Human Genetics, 108(12): 2336–2353, 2021.</i>
		</p>
		
		<p class="bibitem">
		Ma, S., Dalgleish, J., Lee, J., Wang, C., <b>Liu, L.</b>, Gill, R., Buxbaum, J. D., Chung, W. K., Aschard, H., Silverman, E. K., Cho, M. H., He, Z. and Ionita-Laza, I. 
		<a href="https://doi.org/10.1073/pnas.2105191118">Powerful gene-based testing by integrating long-range chromatin interactions and knockoff genotypes.</a> 
		<i>Proceedings of the National Academy of Sciences USA, 118(47): e2105191118, 2021.</i>
		</p>
		
		<p class="bibitem">
		He, Z., <b>Liu, L.</b>, Wang, C., Le Guen, Y., Lee, J., Gogarten, S., Lu, F., Montgomery, S., Tang, H., Silverman, E., Cho, M., Greicius, M. and Ionita-Laza, I. 
		<a href="https://doi.org/10.1038/s41467-021-22889-4">Identification of putative causal loci in whole-genome sequencing data via knockoff statistics.</a> 
		<i>Nature Communications, 12(1): 3152, 2021.</i>
		</p>
		
		<p class="bibitem">
		He, Z., <b>Liu, L.</b>, Wang, K. and Ionita-Laza, I. 
		<a href="https://www.nature.com/articles/s41467-018-07349-w">A semi-supervised approach for predicting cell-type specific functional consequences of non-coding variation using MPRAs.</a> 
		<i>Nature Communications, 9(1): 5199, 2018.</i>
		</p>
		
		<p class="bibitem">
		<b>Liu, L.</b>, Li, D. and Wong, W. H. 
		<a href="https://proceedings.neurips.cc/paper/2017/hash/f55cadb97eaff2ba1980e001b0bd9842-Abstract.html">Convergence rates of a partition based Bayesian multivariate density estimation method.</a> 
		<i>Advances in Neural Information Processing Systems (NIPS) 30: 4738–4746, 2017.</i>
		</p>
		
		<p class="bibitem">
		Ye, C., <b>Liu, L.</b>, Wang, X. and Zhang, X. 
		<a href="https://doi.org/10.1007/s11460-011-0148-9">Observations on potential novel transcripts from RNA-Seq data.</a> 
		<i>Frontiers of Electrical and Electronic Engineering in China, 6(2): 275–282, 2011.</i>
		</p>


</div>


    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script src="../../dist/js/bootstrap.min.js"></script>
  </body>
</html>
